Vegetarian-Food-Finder/Controller/SearchViewController.swift

```diff
@@ -9,16 +9,32 @@ import UIKit
 final class SearchViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, UISearchBarDelegate, UISearchResultsUpdating {
     @IBOutlet var tableView: UITableView!
     let searchController = UISearchController()
-    var resturantList: [Resturant] = []
-    var filteredRestaurantsList: [Resturant] = []
+    var restaurantList: [Business] = [] {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    var filteredRestaurantsList: [Business] = []
```

Comment:
If you are filtering the restaurant locally, consider only having one source of truth.
You can use a computed property to get the filtered elements instead of storing them